centreon-dsm/www/modules/centreon-dsm/core/configuration/services/DB-Func.php

```
@@ -477,9 +477,7 @@ function multiplePoolInDB($pool = [], $nbrDup = [])
                         $pearDB->executePreparedQuery($statement, $parameters, true);
                         $pearDB->closeQuery($statement);
 
-                        $statement = $pearDB->executeQuery('SELECT MAX(pool_id) FROM `mod_dsm_pool`');
-                        $cmd_id = $pearDB->fetch($statement);
-                        $pearDB->closeQuery($statement);
+                        $cmd_id = ['MAX(pool_id)' => (int) $pearDB->lastInsertId()];
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$cmd_id'. (undefined)

(UnusedLocalVariable)